api/views.py

```diff
@@ -50,6 +53,45 @@ def unauthorized(message="You must be logged in"):
     return JsonResponse({"statusCode": 401, "error_message": message}, status=401)
 
 
+@csrf_exempt
+def upload_image(request):
```

Comment:
- Restrict with logged-in users only
- Track these uploads (Suggestion)
  - Maybe create a table with a columns file path, created by, created at

Reply:
Maybe we can use images_upload_handler to provide CSRF+Auth
https://www.tiny.cloud/docs/demo/local-upload/#